src/functionaltests/java/com/ericsson/ei/threadingAndWaitlistRepeat/ThreadingAndWaitlistRepeatSteps.java

```diff
@@ -79,7 +78,7 @@ public void no_event_is_aggregated() throws Throwable {
     public void the_waitlist_will_try_to_resent_the_events_at_given_time_interval() throws Throwable {
```

Comment:
Should be "the_waitlist_will_try_to_resend_the_events_at_given_time_interval", i.e. resend instead of resent